shaded/client/pom.xml

```diff
@@ -283,6 +282,7 @@
                     <exclude>org/w3c/dom/**/*</exclude>
                     <exclude>org/xml/sax/*</exclude>
                     <exclude>org/xml/sax/**/*</exclude>
+                    <exclude>org/rocksdb/**/*</exclude>
```

Comment:
Rocksdb package is not belong to JDK. Maybe we could move `<exclude>org/rocksdb/**/*</exclude>` to another place?

Reply:
Thank you @JySongWithZhangCe !  good call!  just added a comment to make it looks like as a separate section

Reply:
LGTM！Thinks for your contribution!